<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tsearch_dni</class>
 <widget class="QMainWindow" name="tsearch_dni">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
border: 3px solid rgb(0, 0, 127);
background-color: rgb(170, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}
QLabel{
border: none;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}
QComboBox{
border: none;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}
QLineEdit{
border: none;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_des">
         <property name="styleSheet">
          <string notr="true">border:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_name_s">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid rgb(0, 0, 127);
border-radius:0px;</string>
            </property>
            <item>
             <property name="text">
              <string>Nombres</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Apellidos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nro celular</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_data_search">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_description">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Liste de nombres o apellidos que coinciden</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QListWidget" name="listWidget_show_s">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid rgb(0, 0, 127);
border-radius:1px;
background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>Hello world</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Never mind</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_seleccionado_dni">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Se ha seleccionado el DNI :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QFrame" name="horizontalFrame">
         <property name="styleSheet">
          <string notr="true">QFrame{
border: none;
}
QPushButton{
border: 1px solid rgb(0, 0, 127);
border-radius:10px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(170, 85, 255);
}

QPushButton:hover{
background-color: rgb(170, 255, 255);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>15</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_get_dni">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Obtener el DNI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_decline">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_superior">
         <property name="styleSheet">
          <string notr="true">QFrame{
border: none;
}</string>
         </property>
         <layout class="QHBoxLayout" name="layout_principal">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="lbl_title">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Mostrar datos</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ampliar">
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(170, 170, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover{
	
	background-color: rgb(170, 255, 255);
}</string>
            </property>
            <property name="text">
             <string>■</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close_search">
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover{
	background-color: rgb(170, 0, 255);
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
